src/test/java/io/strimzi/test/container/StrimziKafkaContainerIT.java

```diff
@@ -312,6 +312,7 @@ void testGetProxyWithNoContainer(final String imageName) {
                 .waitForRunning();
         systemUnderTest.start();
         assertThrows(IllegalStateException.class, () -> systemUnderTest.getProxy());
+        systemUnderTest.stop();
```

Comment:
Wouldn't be better to put it into `afterEach` ?

Reply:
> Wouldn't be better to put it into `afterEach` ?

Some tests doesn't to start but I think that we can solve it like `if sut != null -> sut.stop()`.

Reply:
Ye that was exactly the proposed solution I was thinking about. It is just an idea, if you think it will be better to just add it into one test I am fine with it.

Reply:
Like second option would be to re-write all of those tests to `try-with-resources` as this class implements AutoClosable.

Reply:
Update both for ZK and KRaft suite with afterEach.

Reply:
so should this be removed or it should be kept?